### DISCUSSION TITLE
Battery profile for PIS-1191

### DISCUSSION THREAD

**Discussion by kvanwezemael** [PiSupply/PiJuice#860] [Q&A]
Dear,

I was wondering what the correct battery profile is for the 600mAh PIS-1191 battery.

Using the PJLIPO_600 profile seems to yield wrong battery temperature ratings, so my guess is that another value for the NTC resistor (and maybe others) is needed.

With kind regards,

Kris

**Comment by PoliceMF** [comment]
Hello,

I just create my account to say that your not alone Kris. I'm having the same problem.

Pijuice, Could you please tell me  where to find the correct battery profiles for the official Pijuice batteries.
In particular I'm also searching for the PIS-1191 battery parameters

Thank you,

**Comment by tvoverbeek** [comment]
What wrong temperatures are you getting? Too hot ot too cold?
It could be there is a batch of PIS-1191 with an other NTC.
There was a smilar issue with the 12000mAh battery.
You could try the following:
Use a Cutom profile based on the PJLIPO_600 but with the following values for the NTC:
ntcB 5200
ntcResistance 42000
(These are the values used in the alternate 12000 mAh profile).
Alternatively contact Pi-SUpply directly: https://uk.pi-supply.com/pages/contact-us
I am not employed by Pi-Supply, just a volunteer.

**Reply by kvanwezemael** [reply]
Hello tvoverbeek,

Thank you for taking time to reply.

With the PJLIPO_600 profile selected, status is reporting -7°C while my ambient thermometer shows 27°C
Measuring the NTC at 27°C yields about 44.2 KOhm, so indeed this is no standard 10K NTC
Helas, entering ntcResistance and ntcB as mentioned above yields in status 47°C while 27 degrees is measured

With kind regards,

Kris

**Comment by tvoverbeek** [comment]
So the NYTC resistance value of 42000 mentioned earlier looks reasonable.
If you disable the NTC measurement the firmware assumes a default temperature of 25‡C.
@mmilann Do you know the values for the alternative NTC in the 600 mAh battery?
